cdap-app-templates/cdap-etl/cdap-etl-realtime-app/pom.xml

```diff
@@ -52,6 +52,11 @@
       <version>${project.version}</version>
     </dependency>
     <dependency>
+      <groupId>co.cask.cdap</groupId>
+      <artifactId>cdap-etl-common</artifactId>
```

Comment:
we want to be able to use ``StructuredRecordStringConverter`` in ``ETLWorker``, so adding this dependency